src/components/casehistory/CaseHistoryList.js

```diff
@@ -150,9 +149,16 @@ const CaseHistoryList = ({
               <InfoItem modal={modal}>{`File Date: ${formattedFileDate}`}</InfoItem>
               <InfoItem modal={modal}>
                 { (psaPermissions && hasBeenUpdated)
-                  ? <PendingChargeStatus pendingCharges>Updated</PendingChargeStatus>
+                  ? <Tag mode="danger">Updated</Tag>
                   : null}
               </InfoItem>
+              <InfoItem modal={modal}>
```

Comment:
If `caseStatus === 'Terminated'` is false, we end up with 

```
<InfoItem modal={modal}>
  {null}
</InfoItem>
```

Do you need to render `InfoItem` if it has `null` children?